src/components/CostEstimationContent/CostOperation.tsx

```diff
@@ -139,7 +142,8 @@ export const CostOperation: FunctionComponent = () => {
         },
       },
     ];
-  }, [dateTime, gwei, maticGwei, maticPrice, maticPriceFactor, price, priceFactor]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [gwei, maticGwei]);
```

Comment:
slapping ignoring of lint errors doesnt make this better you know.

im trying to get you to realise that this cost data function can be constructed with these 2 variables only.